students/tri_nguyen/lesson04/trigrams.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  28-Feb-2018
+# ------------------------------------------- #
+
+import random
+
+
+def get_word_list():
+    ''' this function create a list of words from 
+        sherlock_small.txt '''
+
+    word_list = []
+
+    with open('sherlock_small.txt') as f:
+        for line in f:
+            line.rstrip()
```

Comment:
strings are immutable, so if you want to actually strip you need to assign result: `line = line.rstrip()`